tests/Client/GetLearnerReportClientTest.php

```diff
@@ -169,12 +172,26 @@ public function testAddUsersToGroupThrowsExceptionWhenFatalErrorReturned() {
         $httpClient = new HttpClient(['handler' => $handlerStack]);
         $client->setHttpClient($httpClient);
 
-        // Make the request.
-        self::expectException(SmarterUException::class);
-        self::expectExceptionMessage(
-            'SmarterU rejected the request due to the following error(s): 1: An error has occurred'
-        );
-        $client->getLearnerReport($query);
+        // Make the request. Because we want to inspect custom exception
```

Comment:
Glad for the comment. When I got to this block I was thinking "why not `expectException`?"